models/sessionization/segment_web_page_views__sessionized.sql

```diff
@@ -93,7 +93,20 @@ new_sessions as (
         case
             when period_of_inactivity <= {{var('segment_inactivity_cutoff')}} then 0
             else 1
-        end as new_session
+        end as new_session_inactivity,
+        case
+            when previous_tstamp::DATE != tstamp::DATE then 1
+            else 0
+        end as new_session_midnight,
+        case
+           when referrer_host ilike '%beautypiehelp.zendesk%' or referrer_host = 'beautypie.com' then 0
+           when referrer_host is NULL and page_view_number = 1 then 1
+           when referrer_host is NULL then 0
+           when referrer_host = '' then 0 -- TBC
```

Comment:
TBC?

Reply:
yahh it's so I don't forget. There's apparently some sort of fix going on with '' atm but in the meantime this is how we've agreed to define it 🤷‍♀️